src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/render/ModuleJumpEffect.kt

```diff
@@ -0,0 +1,71 @@
+package net.ccbluex.liquidbounce.features.module.modules.render
+
+import net.ccbluex.liquidbounce.event.PlayerJumpEvent
+import net.ccbluex.liquidbounce.event.WorldRenderEvent
+import net.ccbluex.liquidbounce.event.handler
+import net.ccbluex.liquidbounce.event.repeatable
+import net.ccbluex.liquidbounce.features.module.Category
+import net.ccbluex.liquidbounce.features.module.Module
+import net.ccbluex.liquidbounce.render.drawGradientCircle
+import net.ccbluex.liquidbounce.render.engine.Color4b
+import net.ccbluex.liquidbounce.render.engine.Vec3
+import net.ccbluex.liquidbounce.render.renderEnvironmentForWorld
+import net.ccbluex.liquidbounce.render.utils.shiftHue
+import net.ccbluex.liquidbounce.render.withPosition
+import net.ccbluex.liquidbounce.utils.client.Curves
+import org.apache.commons.lang3.tuple.MutablePair
+
+object ModuleJumpEffect : Module("JumpEffect", Category.RENDER) {
+    private val endRadius by floatRange("EndRadius", 0.15F..0.8F, 0F..3F)
+
+    private val innerColor by color("InnerColor", Color4b(0, 255, 4, 0))
+    private val outerColor by color("OuterColor", Color4b(0, 255, 4, 89))
+
+    private val animCurve by curve("AnimCurve", Curves.EASE_OUT)
+
+    private val hueOffsetAnim by int("hueOffsetAnim", 63, -360..360)
```

Comment:
To make it fit with the capitalisation of the rest, please name it "HueOffsetAnim".

Reply:
Yes, I missed that